core/src/common/resources/lambdas/cdk-deployer/reportbuild.js

```diff
@@ -0,0 +1,84 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+const respond = async function(event, context, responseStatus, responseData, physicalResourceId, noEcho) {
```

Comment:
is it even bundled then?

Reply:
this one is not used. We need to change the PreBundledFunction and then we can use this lambda code